data/monster/monsters/demon.xml

```diff
@@ -16,6 +16,18 @@
 		<flag staticattack="70" />
 		<flag runonhealth="0" />
 	</flags>
+	<bestiary>
+		<bestiary class="Demon" />
+		<bestiary race="Demon" /> <!-- or Demon etc-->
+		<bestiary raceId="35" />
+		<bestiary firstUnlock ="100" />
+		<bestiary secondUnlock ="500" />
+		<bestiary finishUnlock ="2500" />
+		<bestiary charmPoints ="50" />
+		<bestiary stars ="4" />
+		<bestiary occurrence ="1" />
+		<bestiary locations = "Test locations" />
+	</bestiary>
```

Comment:
This should be a single tag:
```xml
<bestiary class="..." race="..." />
```

Also, I believe it would read better if we used the same names as bosstiary: `firstUnlock`->`prowess`, `secondUnlock`->`expertise`, `lastUnlock`-> `mastery`

Reply:
> This should be a single tag:
> 
> ```
> <bestiary class="..." race="..." />
> ```
> 
> Also, I believe it would read better if we used the same names as bosstiary: `firstUnlock`->`prowess`, `secondUnlock`->`expertise`, `lastUnlock`-> `mastery`

I think the race could be in the header, considering that it is also used to identify prey and boosted creature monsters.
as I did here: https://github.com/otland/forgottenserver/pull/3857

Reply:
by the way,
amount of creatures to kill and charm points reward are dependent on rarity and difficulty, they are regular

Reply:
> How is race actually used by the client? Is it able to show any string or should we use defined constants to restrict the use to values the client can understand.

depending on where you send it - races in client have name and outfit hardcoded, though some places allow different monster name/outfit. It's inconsistent.